add siparcs blog post (#384)
xarray-contrib/xarray.dev: src/posts/siparcs_2022/marshall_siparcs_2022.md

The Jupyter Book tutorials each focus on a different remote sensing dataset and contain Jupyter Notebooks detailing common steps of a research workflow. They demonstrate working with global ice velocity measurements derived from a combination of optical and synthetic aperture radar (SAR) satellite imagery as well as SAR radiometric terrain corrected (RTC) backscatter imagery. These data make use of cloud-native processing chains and cover a range of cloud-native and traditional data types and the tutorials include examples of reading directly from object storage as well as working with data locally. In choosing these examples, we focused on time series datasets that lend themselves to working with Xarray and provide valuable information about Earth’s surface processes that can be used in a range of climate-related applications such as glacier dynamics, hydrology, hazards, and studies of vegetative cover and extent.

> 

Reviewer: ```suggestion
Here are the two books I created:
 1. [Using Xarray to examine cloud-based glacier surface velocity data](https://e-marshall.github.io/itslive/)

Based on the Inter-mission Time Series of Land Ice Velocity and Elevation ([ITS_LIVE](https://its-live.jpl.nasa.gov/)) dataset.

2. [Sentinel-1 RTC imagery workflows with Xarray](https://e-marshall.github.io/sentinel1_rtc/)

Using Sentinel-1 Radiometric Terrain Corrected (RTC) backscatter datasets processed and hosted by both [Microsoft Planetary Computer](https://planetarycomputer.microsoft.com/dataset/sentinel-1-rtc#overview) and [Alaska Satellite Facility](https://www.arcgis.com/home/item.html?id=3dd8d25559db4ba6aa0e1b6e8cb5d39a)
```

Not sure this will render correctly

<br>
![](https://i.imgur.com/nh5K9AB.png)
*Exploring seasonal variability in Sentinel-1 RTC backscatter using `groupby()` and `FacetGrid`*

> #### Jupyter book tutorials
> 
> ##### 1.[Using Xarray to examine cloud-based glacier surface velocity data](https://e-marshall.github.io/itslive/)
> 
> Based on the Inter-mission Time Series of Land Ice Velocity and Elevation ([ITS_LIVE](https://its-live.jpl.nasa.gov/)) dataset.
> 
> ##### 2.[Sentinel-1 RTC imagery workflows with Xarray](https://e-marshall.github.io/sentinel1_rtc/)
> 
> Using Sentinel-1 Radiometric Terrain Corrected (RTC) backscatter datasets processed and hosted by both [Microsoft Planetary Computer](https://planetarycomputer.microsoft.com/dataset/sentinel-1-rtc#overview) and [Alaska Satellite Facility](https://www.arcgis.com/home/item.html?id=3dd8d25559db4ba6aa0e1b6e8cb5d39a)

Reviewer: ```suggestion

```

## Conclusion

This summer, I got to experience how fun and exciting it is to talk through concepts, work on issues, and problem-solve with people who are passionate about what they do, curious about interesting questions, and selfless in the time and knowledge they invest back into the community. My experience as an Xarray intern and more general community interactions I had both on GitHub and Pangeo left me with a set of role models who I admire for the emphasis they place on sharing knowledge in a constructive, intentional, and community-minded way, with an eye toward developing the tools, but also culture, that will help use data to solve large, important problems now and in the future.

> ## Jupyter book tutorials
> 
> ### 1.[Using Xarray to examine cloud-based glacier surface velocity data](https://e-marshall.github.io/itslive/)
> 
> Based on the Inter-mission Time Series of Land Ice Velocity and Elevation ([ITS_LIVE](https://its-live.jpl.nasa.gov/)) dataset.
> 
> ### 2.[Sentinel-1 RTC imagery workflows with Xarray](https://e-marshall.github.io/sentinel1_rtc/)
> 
> Using Sentinel-1 Radiometric Terrain Corrected (RTC) backscatter datasets processed and hosted by both [Microsoft Planetary Computer](https://planetarycomputer.microsoft.com/dataset/sentinel-1-rtc#overview) and [Alaska Satellite Facility](https://www.arcgis.com/home/item.html?id=3dd8d25559db4ba6aa0e1b6e8cb5d39a)

Reviewer: ```suggestion
```

Duplicated above

The Jupyter Book tutorials each focus on a different remote sensing dataset and contain Jupyter Notebooks detailing common steps of a research workflow. They demonstrate working with global ice velocity measurements derived from a combination of optical and synthetic aperture radar (SAR) satellite imagery as well as SAR radiometric terrain corrected (RTC) backscatter imagery. These data make use of cloud-native processing chains and cover a range of cloud-native and traditional data types and the tutorials include examples of reading directly from object storage as well as working with data locally. In choosing these examples, we focused on time series datasets that lend themselves to working with Xarray and provide valuable information about Earth’s surface processes that can be used in a range of climate-related applications such as glacier dynamics, hydrology, hazards, and studies of vegetative cover and extent.
Here are the two books I created:

> 1.  [Using Xarray to examine cloud-based glacier surface velocity data](https://e-marshall.github.io/itslive/)
> 
> Based on the Inter-mission Time Series of Land Ice Velocity and Elevation ([ITS_LIVE](https://its-live.jpl.nasa.gov/)) dataset.
> 
> 2. [Sentinel-1 RTC imagery workflows with Xarray](https://e-marshall.github.io/sentinel1_rtc/)
> 
> Using Sentinel-1 Radiometric Terrain Corrected (RTC) backscatter datasets processed and hosted by both [Microsoft Planetary Computer](https://planetarycomputer.microsoft.com/dataset/sentinel-1-rtc#overview) and [Alaska Satellite Facility](https://www.arcgis.com/home/item.html?id=3dd8d25559db4ba6aa0e1b6e8cb5d39a)

Reviewer 2: ```suggestion
1.  [Using Xarray to examine cloud-based glacier surface velocity data](https://e-marshall.github.io/itslive/)
 
    Based on the Inter-mission Time Series of Land Ice Velocity and Elevation ([ITS_LIVE](https://its-live.jpl.nasa.gov/)) dataset.

2. [Sentinel-1 RTC imagery workflows with Xarray](https://e-marshall.github.io/sentinel1_rtc/)

   Using Sentinel-1 Radiometric Terrain Corrected (RTC) backscatter datasets processed and hosted by both [Microsoft Planetary Computer](https://planetarycomputer.microsoft.com/dataset/sentinel-1-rtc#overview) and [Alaska Satellite Facility](https://www.arcgis.com/home/item.html?id=3dd8d25559db4ba6aa0e1b6e8cb5d39a)
```

> <br>
> ![](https://i.imgur.com/nh5K9AB.png)
> *Exploring seasonal variability in Sentinel-1 RTC backscatter using `groupby()` and `FacetGrid`*

Reviewer 2: ```suggestion
<br />

![](https://i.imgur.com/nh5K9AB.png)
*Exploring seasonal variability in Sentinel-1 RTC backscatter using `groupby()` and `FacetGrid`*
```

---

> title: 'Reflection on 2022 NCAR summer internship: "Leveraging Xarray for reproducible and scalable analysis of remote sensing data in the cloud"'

Reviewer 2: Could the title be shortened?

<img width="1104" alt="Screenshot 2022-10-28 at 11 07 24 AM" src="https://user-images.githubusercontent.com/13301940/198693376-9e3f71ba-e6a3-464f-adec-33e63ac7a0a0.png">

Reviewer: ```suggestion
title: 'Reflections on my NCAR summer internship working with Xarray and satellite data"'
```

Hmm.. not the biggest fan of this suggestion

Author: thanks @andersy005 and @dcherian ! happy to make more changes if needed as well

> # Reflection on 2022 NCAR summer internship: "Leveraging Xarray for reproducible and scalable analysis of remote sensing data in the cloud"

Reviewer 2: ```suggestion
---
title: 'Reflection on 2022 NCAR summer internship: "Leveraging Xarray for reproducible and scalable analysis of remote sensing data in the cloud"'
date: '2022-10-27'
authors:
  - name: Emma Marshall
    github: e-marshall
---
```
